scripts/kube-init.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
```

Comment:
How/where do we use this file? If it is a manual run, it would be nice if we have an example here.